Makefile

```diff
@@ -320,11 +332,22 @@ TEST_DEPS := $(sort $(TEST_DEPS))
 .PHONY: all
 all: $(BINS)
 
+.PHONY: set_dynamic_options
+set_dynamic_options: $(TOML)
+	$(eval DYNAMIC_OPTIONS = -fPIC)
```

Comment:
What is this for? Would I ever call `make set_dynamic_options`?

Reply:
I used it as a way to require this statement as a prerequisite to BACKUP_DYNAMIC and RESTORE_DYNAMIC when running `make shared` you wouldn't run `make set_dynamic_options` as a user.

Reply:
Added a comment

Reply:
seems weird since it isn't really building anything. Would it not always run when running `make shared`?